### PR TITLE
Add vendored getTileIndices

### PR DESCRIPTION
**Pull request by ilan-gold**
I could not get this to run locally, but I think this is the meat and potatoes of it.

### CODE REVIEW

**Comment by manzt** [comment]
I see there is no longer an array of zoom levels for the same region. Isn't that what we want to iterate through? Maybe I'm confused. Otherwise, this looks good to me.

File: src/config.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
       "regions": [
         { 
           "id": "some-unique-id",
-          "left": 0, 
-          "top": 0, 
-          "zoomLevels": [0, 1, 2],
```
Comment (manzt) [inline src/config.json:24]: for a particular region, don't we want to calculate the tiles needed at multiple zoom levels? Is that being accomplished somewhere?

**Review by manzt — APPROVED** [review]

File: src/config.json (hunk 1 above)
Comment (ilan-gold) [reply src/config.json:24]: Yes there is a for loop in `getTileIndices` - I though that that was the desired behavior, to provide one zoom level and get all the higher resolution ones "under" it.

Comment (ilan-gold) [reply src/config.json:24]: https://github.com/manzt/viv-tile-benchmark/pull/1/files#diff-6cf089419f06bbfa687fbcf29710f534R68-R83

Comment (manzt) [reply src/config.json:24]: Ah sorry. Is that a part of the vendored code or a function you added? If added, maybe move outside of the module.

**Comment by manzt** [comment]
```bash
❯ cat chrome_http1.csv
file,format,compression,zoom,tileSize,regionId,yCoord,xCoord,height,width,numChannels,tileId,startTime,endTime
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--6,590.1550000016869,972.845000000234
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--5,592.3650000004272,1537.8950000012992
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--4,593.1049999999232,1696.8600000000151
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--3,594.1850000017439,1671.3349999990896
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--2,594.7950000008859,1265.5699999995704
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,1,0-0--1,595.3350000017963,825.7149999990361
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--6,1700.345000001107,1912.7150000022084
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--5,1700.8800000003248,2382.740000000922
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--4,1701.3450000013108,2499.4750000005297
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--3,1701.7550000018673,2449.4550000017625
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--2,1702.1700000004785,2154.5550000009825
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,1,0-0--1,1702.5849999990896,1797.2799999988638
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--6,2501.2950000018463,2926.755000000412
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--5,2502.250000001368,3779.3950000013865
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--4,2502.889999999752,4026.754999998957
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--3,2504.059999999299,3964.654999999766
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--2,2504.7650000014983,3426.484999999957
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,2,0-0--1,2505.6250000016007,2654.1700000016135
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--6,4028.425000000425,4448.585000001913
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--5,4029.239999999845,5137.634999999136
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--4,4030.044999999518,5388.605000000098
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--3,4030.784999999014,5343.0350000016915
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--2,4032.054999999673,4764.0749999991385
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,2,0-0--1,4032.9750000018976,4238.345000001573
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--6,5390.56999999957,5747.42999999944
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--5,5391.855000001669,6145.035000001371
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--4,5392.845000002126,6212.8650000013295
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--3,5393.755000000965,6190.555000001041
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--2,5394.93500000026,5965.390000001207
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,3,0-0--1,5396.110000001499,5510.949999999866
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--6,6214.4699999989825,6311.434999999619
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--5,6223.860000001878,6683.069999999134
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--4,6224.900000001071,6759.229999999661
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--3,6225.8499999989,6740.825000000768
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--2,6226.795000002312,6528.880000001664
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,3,0-0--1,6227.745000000141,6308.254999999917
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--6,6761.0400000012305,6943.334999999934
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--5,6762.380000000121,7378.534999999829
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--4,6763.549999999668,7500.530000001163
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--3,6764.79499999914,7492.3200000012
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--2,6766.090000000986,7261.020000001736
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,300,400,4,0-0--1,6767.30999999927,6864.260000002105
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--6,7502.2400000016205,7588.380000001052
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--5,7511.210000000574,8140.535000002274
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--4,7512.460000001738,8217.854999998963
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--3,7513.654999998835,8199.730000000272
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--2,7514.885000000504,7946.674999999232
original_file_name.img,zarr,blosc,-5,1024,some-unique-id,0,0,1000,1000,4,0-0--1,7516.169999998965,7625.485000000481
```

I only see one zoom level logged in the output, are you sure all the higher zoom levels are being computed?

**Comment by ilan-gold** [comment]
@manzt This should be fixed now.  It was a reporting bug.  I also noticed another issue but fixed it.

**Review by manzt — APPROVED** [review]

**Pull request merged**